include/quicr/name.h

```diff
@@ -152,58 +156,77 @@ class Name
 
         if (hex_value.length() > sizeof(Name))
         {
-            _hi = hex_to_uint<uint_type>(hex_value.substr(0, hex_value.length() - sizeof(Name)));
-            _lo = hex_to_uint<uint_type>(hex_value.substr(hex_value.length() - sizeof(Name), sizeof(Name)));
+            _hi = hex_to_uint<uint_t>(hex_value.substr(0, hex_value.length() - sizeof(Name)));
+            _lo = hex_to_uint<uint_t>(hex_value.substr(hex_value.length() - sizeof(Name), sizeof(Name)));
         }
         else
         {
             _hi = 0;
-            _lo = hex_to_uint<uint_type>(hex_value.substr(0, hex_value.length()));
+            _lo = hex_to_uint<uint_t>(hex_value.substr(0, hex_value.length()));
         }
-    }
 
-    constexpr Name& operator=(std::string_view hex) { return *this = Name(hex); }
+        if (align_left) *this <<= (sizeof(Name) - (hex_value.length() / 2)) * 8;
```

Comment:
why divide by 2 here?

Reply:
Because a hex string will have 2 characters for each byte